test-setup-config.js

```diff
@@ -0,0 +1,23 @@
+import config from '@plone/volto/registry';
+
+config.set('settings', {
+  publicURL: 'http://localhost:3000',
+  italiaThemeViewsConfig: {
+    imagePosition: 'afterHeader', // possible values: afterHeader, documentBody
+  },
+  imageScales: {
+    listing: 16,
+    icon: 32,
+    tile: 64,
+    thumb: 128,
+    mini: 200,
+    midi: 300,
+    preview: 400,
+    teaser: 600,
+    large: 800,
+    larger: 1000,
+    great: 1200,
+    huge: 1600,
+  },
+  ...config.settings,
```

Comment:
Non sono sicuro, ma ho idea che fare così sia equivalente a non fare niente, perché le chiavi già esistenti in `config.settings` dovrebbero sovrascrivere quelle che setti manualmente qui.. a cosa serve questo setup?

Reply:
hai ragione, ma in realtà ho aggiunto quelle che mancavano, non ho avuto bisogno di modificarne.